### PR TITLE
app-i18n/opencc: add python use flag

### PR DESCRIPTION
**Pull request by blackteahamburger**
Closes: https://bugs.gentoo.org/938212

References Arch's [PKGBUILD](https://gitlab.archlinux.org/archlinux/packaging/packages/opencc/-/blob/main/PKGBUILD).

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: app-i18n/opencc/opencc-1.1.7.ebuild [hunk 1]
```diff
@@ -14,18 +18,27 @@ S="${WORKDIR}/OpenCC-ver.${PV}"
 
 LICENSE="Apache-2.0"
 SLOT="0/1.1"
-KEYWORDS="~amd64 ~arm64 ~hppa ~loong ~ppc ~ppc64 ~riscv ~sparc ~x86"
-IUSE="doc test"
+KEYWORDS="~amd64 ~arm64 ~ppc ~ppc64 ~riscv ~x86"
```
Comment (liangyongxiang) [inline app-i18n/opencc/opencc-1.1.7.ebuild:21]: There should be no reason to drop part of the keyword here.

**Comment by liangyongxiang** [comment]
After ebuild adds the new use, it should add revison.

File: app-i18n/opencc/opencc-1.1.7.ebuild (hunk 1 above)
Comment (blackteahamburger) [reply app-i18n/opencc/opencc-1.1.7.ebuild:21]: app-i18n/opencc[python] has app-admin/chrpath BDEPEND.

`KEYWORDS` of app-admin/chrpath:

`KEYWORDS="amd64 arm ~arm64 ~mips ppc ppc64 ~riscv x86 ~amd64-linux ~x86-linux ~x64-solaris"`

[Devmanual](https://devmanual.gentoo.org/keywording/#equal-visibility-requirement):

> An ebuild must not depend upon any package that is of a lower keyword level than itself. For example, if foo-1.2 depends upon bar-1.2, and bar-1.2 is ~x86, then foo-1.2 must not be marked stable on x86 unless bar-1.2 is also stabilised.

So ~hppa ~loong are dropped.

**Comment by blackteahamburger** [comment]
> After ebuild adds the new use, it should add revison.

[Devmanual](https://devmanual.gentoo.org/general-concepts/ebuild-revisions/):

> Examples of changes that can be done without a revision bump are:
> ...
> - adding a new USE flag if it controls a USE-dependency where the functionality was hard-disabled in the build system before,

File: app-i18n/opencc/opencc-1.1.7.ebuild (hunk 1 above)
Comment (dlan17) [reply app-i18n/opencc/opencc-1.1.7.ebuild:21]: given these keywords are dropped, I would like to see a revision bump (I can handle this while merging)

can you also file a keyword request for these ARCHs?

Comment (blackteahamburger) [reply app-i18n/opencc/opencc-1.1.7.ebuild:21]: See https://bugs.gentoo.org/938577.

**Pull request closed without merging**